test/powershell/Modules/Microsoft.PowerShell.Security/ExecutionPolicy.Tests.ps1

```diff
@@ -910,6 +938,46 @@ ZoneId=$FileType
             foreach($testScript in $testScripts) {
                 Test-AllSignedExecutionPolicy $testScript $error
             }
+            $TestTypePrefix = "Test 'AllSigned' execution policy."
+
+            $error = "System.Management.Automation.PSSecurityException"
+            $testData = @(
+                @{
+                    testScript = $LocalUnsignedModule
+                    expected = $null
+                    error = $error
+                }
+                @{
+                    testScript = $LocalSignedModule
+                    expected = $null
+                    error = $null
+                }
+            )
+
+            It "$TestTypePrefix Running <testScript> Module should return <error>" -TestCases $testDate  {
+
+                $scriptName = $testScript
+
+                $exception = $null
+                try
+                {
+                    & $scriptName
+                }
+                catch
+                {
+                    $exception = $_
```

Comment:
change to use `ShouldBeErrorId`?

Reply:
Sorry, I accidentally pushed an incomplete commit.  I have updated the PR.